src/components/treemap.tsx

```diff
@@ -80,19 +80,48 @@ export default class Treemap extends React.Component<
       width={d.x1 - d.x0}
       height={d.y1 - d.y0}
     />
-  ));
+  ))
+
+    sizingProperties = d => {
+        const [ minValue, maxValue ] = this.props.fontSize;
+        const x = max([0, Math.floor(((d.x1 - d.x0) - 8))]);
+        const y = max([0, Math.floor(((d.y1 - d.y0) - 8))]);
+        const label = d.data.location_name;
+        const width = stringWidth(label, 12) / 12;
```

Comment:
What's `12`? And what's `8` above here? I think it'd be cool to have these listed as CONSTANTS at the top of the file, or in a separate imported module if it get too cluttered. Thoughts?

Reply:
Agreed- this was stuff from Evan's code that I left in so I'm totally open to making it prettier